LPS-162388 Simple solution. PR for tests

ricardocousolr opened the pull request:
tests

ricardocousolr committed:
LPS-162388 Immediately return empty link if user should not have access

If a notification feed entry's applicable is false it will show marked
as read, have no link and no actions. It was meant for ephemeral
notifications associated to workflow tasks that were withdrawn.

This is too stringent for notifications meant to have no link to another
view but still have the option to mark as read and delete.

For these, applicable should be true and link should be empty. This way
the notification is marked unread, has actions and is given generated
link that doesn't go anywhere but marks the notification read.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

ricardocousolr commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/5463/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5463/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 28 out of 28 jobs passed</h3><h3>:x: ci:test:relevant - 82 out of 88 jobs passed in 3 hours 56 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a8d5e3f19cc9420135bdb44f779952f11a4c9b31">a8d5e3f19cc9420135bdb44f779952f11a4c9b31</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/fb2f2d6cc318ef108e21bfc615b4e09dcbff907e">fb2f2d6cc318ef108e21bfc615b4e09dcbff907e</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2070187173">EE Development Acceptance (master) - 4045 - 2022-11-22[00:17:39]</a></p><details><summary><strong>ci:test:stable - 28 out of 28 jobs PASSED</strong></summary><details><summary><strong>28 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 82 out of 88 jobs PASSED</strong></summary><div><h4>6 Failed Jobs:</h4><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/5463/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116877/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327664/">empty-osgi-core-dir-jdk8/0/0</a></li></ul></li><li>empty-temp-dir-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/350975/">empty-temp-dir-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/120553/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/286983/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/124654/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/133316/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/129569/">functional-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/106174/">functional-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/132397/">functional-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/176513/">functional-tomcat90-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/113129/">functional-tomcat90-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/286985/">functional-tomcat90-mysql57-jdk8/0/5</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116876/">functional-tomcat90-mysql57-jdk8/0/6</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308740/">functional-tomcat90-mysql57-jdk8/0/7</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327668/">functional-tomcat90-mysql57-jdk8/0/8</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327667/">functional-tomcat90-mysql57-jdk8/0/9</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337605/">functional-tomcat90-mysql57-jdk8/0/10</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/119956/">functional-tomcat90-mysql57-jdk8/0/11</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337606/">functional-tomcat90-mysql57-jdk8/0/12</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116871/">functional-tomcat90-mysql57-jdk8/0/13</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/130480/">functional-tomcat90-mysql57-jdk8/0/14</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/133317/">functional-tomcat90-mysql57-jdk8/0/15</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/113128/">functional-tomcat90-mysql57-jdk8/0/16</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/286984/">functional-tomcat90-mysql57-jdk8/0/17</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116875/">functional-tomcat90-mysql57-jdk8/0/18</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/130281/">functional-tomcat90-mysql57-jdk8/0/19</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/319807/">functional-tomcat90-mysql57-jdk8/0/20</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/130481/">functional-tomcat90-mysql57-jdk8/0/21</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/111766/">functional-tomcat90-mysql57-jdk8/0/22</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/111767/">functional-tomcat90-mysql57-jdk8/0/23</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/119958/">functional-tomcat90-mysql57-jdk8/0/24</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298732/">functional-tomcat90-mysql57-jdk8/1/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298731/">functional-tomcat90-mysql57-jdk8/1/1</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/131851/">functional-tomcat90-mysql57-jdk8/1/2</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/126277/">functional-tomcat90-mysql57-jdk8/1/3</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/126276/">functional-tomcat90-mysql57-jdk8/1/4</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298734/">functional-tomcat90-mysql57-jdk8/1/5</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282105/">functional-tomcat90-mysql57-jdk8/1/6</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/263300/">functional-tomcat90-mysql57-jdk8/1/7</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312592/">functional-tomcat90-mysql57-jdk8/1/8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/119957/">functional-tomcat90-mysql57-jdk8/1/9</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312593/">functional-tomcat90-mysql57-jdk8/1/10</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327665/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/117748/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/117747/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282106/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/131852/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/319806/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311960/">js-unit-jdk8/0/0</a></li></ul></li><li>lpkg-controller-jdk8<ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/133315/">lpkg-controller-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337607/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/132396/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/350973/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282104/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/130313/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/263299/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/120554/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/350974/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/126275/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/131850/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116872/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116874/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337609/">modules-compile-jdk8/0/12</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298733/">modules-compile-jdk8/0/13</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/332888/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311962/"><strike>modules-integration-mysql57-jdk8/0/1</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311961/"><strike>modules-integration-mysql57-jdk8/0/2</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116878/">modules-integration-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/176512/"><strike>modules-integration-mysql57-jdk8/0/4</strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327666/"><strike>modules-integration-mysql57-jdk8/0/5</strike></a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282108/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308741/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/350976/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/286381/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312594/"><strike>modules-unit-jdk8/0/0</strike></a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/119313/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337608/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116873/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/298730/">patching-tool-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/282107/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/116870/">portal-license-smoke-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/124655/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/176511/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311959/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308739/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/263301/">unit-jdk8_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>82 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5463/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312594//consoleText">modules-unit-jdk8/0/0</a><ol><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312594/testReport/com.liferay.portal.workflow.task.web.internal.notifications/WorkflowTaskUserNotificationHandlerTest">com.liferay.portal.workflow.task.web.internal.notifications.WorkflowTaskUserNotificationHandlerTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2066048504/history">Failed 0 of last 25</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312594//testReport/com.liferay.portal.workflow.task.web.internal.notifications/WorkflowTaskUserNotificationHandlerTest/testIsApplicable">testIsApplicable</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312594//testReport/com.liferay.portal.workflow.task.web.internal.notifications/WorkflowTaskUserNotificationHandlerTest/testValidWorkflowTaskIdShouldReturnLink">testValidWorkflowTaskIdShouldReturnLink</a></li></ol></div></details></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at fb2f2d6cc318ef108e21bfc615b4e09dcbff907e:</strong></summary><ol><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311962//consoleText">modules-integration-mysql57-jdk8/0/1</a><ol><li><details><summary><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311962/testReport/com.liferay.portal.workflow.kaleo.internal.runtime.integration.test/WorkflowTaskManagerImplTest">com.liferay.portal.workflow.kaleo.internal.runtime.integration.test.WorkflowTaskManagerImplTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2066023655/history">Failed 25 of last 25</a></summary><div><ol><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311962//testReport/com.liferay.portal.workflow.kaleo.internal.runtime.integration.test/WorkflowTaskManagerImplTest/testSearchWorkflowTasksWhenThereIsAnUnregisteredHandler">testSearchWorkflowTasksWhenThereIsAnUnregisteredHandler</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311962/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311962//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311961//consoleText">modules-integration-mysql57-jdk8/0/2</a><ol><li><details><summary><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311961/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311961//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/176512//consoleText">modules-integration-mysql57-jdk8/0/4</a><ol><li><details><summary><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/176512/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/176512//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327666//consoleText">modules-integration-mysql57-jdk8/0/5</a><ol><li><details><summary><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327666/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327666//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li></ol></details></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/5463/">test-portal-acceptance-pullrequest(master)#5463</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5463/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/ricardocousolr/liferay-portal/pull/66">ricardocousolr#66</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2070419909">[master] ci:test:relevant - ricardocousolr &gt; ricardocousolr - PR#66 - 2022-11-22[04:19:23]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/13990/">publish-testray-report#13990</a><br/></div></div>

ricardocousolr committed:
LPS-162388 Split test depending on whether user is allowed or not

Valid workflow tasks can still have no link as expected in user_notification_entry.jspf;
in those cases the current url is set as the link

ricardocousolr committed:
LPS-162388 Remove unnecessary test for applicable which returns true as default

The same logic is now tested inside
testValidWorkflowTaskIdAllowedUserShouldReturnLink
and
testValidWorkflowTaskIdNotAllowedUserShouldReturnBlankLink

ricardocousolr committed:
LPS-162388 Auto SF

ricardocousolr commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/1862/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1862/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 28 out of 28 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 83 out of 87 jobs passed in 2 hours 11 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/2827bb50cedade75335e2127b92d20b698469bf9">2827bb50cedade75335e2127b92d20b698469bf9</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/2827bb50cedade75335e2127b92d20b698469bf9">2827bb50cedade75335e2127b92d20b698469bf9</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2073120807">EE Development Acceptance (master) - 4048 - 2022-11-23[07:36:40]</a></p><details><summary><strong>ci:test:stable - 28 out of 28 jobs PASSED</strong></summary><details><summary><strong>28 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 82 out of 87 jobs PASSED</strong></summary><div><h4>5 Failed Jobs:</h4><ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/1862/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/289364/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/314605/">empty-osgi-core-dir-jdk8/0/0</a></li></ul></li><li>empty-temp-dir-jdk8<ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/131186/">empty-temp-dir-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301335/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/139523/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330350/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330343/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/134673/">functional-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/131187/">functional-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301338/">functional-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/130528/">functional-tomcat90-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/314607/">functional-tomcat90-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/340163/">functional-tomcat90-mysql57-jdk8/0/5</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/335694/">functional-tomcat90-mysql57-jdk8/0/6</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301339/">functional-tomcat90-mysql57-jdk8/0/7</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/265553/">functional-tomcat90-mysql57-jdk8/0/8</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/322546/">functional-tomcat90-mysql57-jdk8/0/9</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330344/">functional-tomcat90-mysql57-jdk8/0/10</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301325/">functional-tomcat90-mysql57-jdk8/0/11</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330345/">functional-tomcat90-mysql57-jdk8/0/12</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301326/">functional-tomcat90-mysql57-jdk8/0/13</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330346/">functional-tomcat90-mysql57-jdk8/0/14</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301327/">functional-tomcat90-mysql57-jdk8/0/15</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/335693/">functional-tomcat90-mysql57-jdk8/0/16</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/314606/">functional-tomcat90-mysql57-jdk8/0/17</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/265552/">functional-tomcat90-mysql57-jdk8/0/18</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/178732/">functional-tomcat90-mysql57-jdk8/0/19</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/139524/">functional-tomcat90-mysql57-jdk8/0/20</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/131352/">functional-tomcat90-mysql57-jdk8/0/21</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/119651/">functional-tomcat90-mysql57-jdk8/0/22</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/353629/">functional-tomcat90-mysql57-jdk8/0/23</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/340164/">functional-tomcat90-mysql57-jdk8/0/24</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330351/">functional-tomcat90-mysql57-jdk8/1/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311484/">functional-tomcat90-mysql57-jdk8/1/1</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/340159/">functional-tomcat90-mysql57-jdk8/1/2</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330353/">functional-tomcat90-mysql57-jdk8/1/3</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/131185/">functional-tomcat90-mysql57-jdk8/1/4</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/340158/">functional-tomcat90-mysql57-jdk8/1/5</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/314604/">functional-tomcat90-mysql57-jdk8/1/6</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/340160/">functional-tomcat90-mysql57-jdk8/1/7</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311487/">functional-tomcat90-mysql57-jdk8/1/8</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301334/">functional-tomcat90-mysql57-jdk8/1/9</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/131351/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311490/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330356/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301337/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330355/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301336/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/340161/">js-unit-jdk8/0/0</a></li></ul></li><li>lpkg-controller-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330342/">lpkg-controller-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301328/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301329/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301330/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301331/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/139522/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/335690/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311488/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/335691/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301333/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311486/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330348/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330352/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311485/">modules-compile-jdk8/0/12</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301332/">modules-compile-jdk8/0/13</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301340/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/322547/"><strike>modules-integration-mysql57-jdk8/0/1</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/314608/"><strike>modules-integration-mysql57-jdk8/0/2</strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/284339/">modules-integration-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/289363/"><strike>modules-integration-mysql57-jdk8/0/4</strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330357/"><strike>modules-integration-mysql57-jdk8/0/5</strike></a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311492/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/335695/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330359/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/335692/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/340162/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330358/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330349/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311483/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330347/">patching-tool-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/178733/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330341/">portal-license-smoke-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311491/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/311489/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330354/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/120850/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/130527/">unit-jdk8_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>82 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1862/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 2827bb50cedade75335e2127b92d20b698469bf9:</strong></summary><ol><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/322547//consoleText">modules-integration-mysql57-jdk8/0/1</a><ol><li><details><summary><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/322547/testReport/com.liferay.portal.workflow.kaleo.internal.runtime.integration.test/WorkflowTaskManagerImplTest">com.liferay.portal.workflow.kaleo.internal.runtime.integration.test.WorkflowTaskManagerImplTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2071662767/history">Failed 25 of last 25</a></summary><div><ol><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/322547//testReport/com.liferay.portal.workflow.kaleo.internal.runtime.integration.test/WorkflowTaskManagerImplTest/testSearchWorkflowTasksWhenThereIsAnUnregisteredHandler">testSearchWorkflowTasksWhenThereIsAnUnregisteredHandler</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/322547/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/322547//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/314608//consoleText">modules-integration-mysql57-jdk8/0/2</a><ol><li><details><summary><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/314608/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/314608//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/289363//consoleText">modules-integration-mysql57-jdk8/0/4</a><ol><li><details><summary><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/289363/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/289363//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330357//consoleText">modules-integration-mysql57-jdk8/0/5</a><ol><li><details><summary><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330357/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/330357//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li></ol></details></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/1862/">test-portal-acceptance-pullrequest(master)#1862</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1862/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/ricardocousolr/liferay-portal/pull/66">ricardocousolr#66</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2073317668">[master] ci:test:relevant - ricardocousolr &gt; ricardocousolr - PR#66 - 2022-11-23[10:13:05]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-39.liferay.com/job/publish-testray-report/4193/">publish-testray-report#4193</a><br/></div></div>
